app/views/partials/yourSelfAssessmentCharges/PaymentHistoryTable.scala.html

```diff
@@ -54,6 +55,7 @@
         case PoaOneDebit     => getMessage("chargeHistory.paymentOnAccount1.heading")
         case PoaTwoDebit     => getMessage("chargeHistory.paymentOnAccount2.heading")
         case BalancingCharge => getMessage("chargeHistory.balancingPayment.heading")
+        case LateSubmissionPenalty => getMessage("chargeHistory.lateSubmissionPenalty.heading")
```

Comment:
same here, if we can move this under something like 
ChargeNameLangHelper.scala